deepinv/utils/plotting.py

```diff
@@ -218,8 +218,8 @@ def plot(
         If this is undesired simply use ``fig = plot(..., show=False, return_fig=True)``
         and plot at your desired location using ``fig.show()``.
 
-    :param list[torch.Tensor], torch.Tensor img_list: list of images to plot or single image.
-    :param list[str] titles: list of titles for each image, has to be same length as img_list.
+    :param list[torch.Tensor], dict[str -> torch.Tensor], torch.Tensor img_list: list of images to plot or single image.
```

Comment:
list of images, single image, or dict of titles: images

Reply:
```suggestion
    :param list[torch.Tensor], dict[str -> torch.Tensor], torch.Tensor img_list: list of images, single image, or dict of titles: images to plot.
```